backend/legacyapi/plan.go

```diff
@@ -190,6 +190,11 @@ const (
 
 	// FeaturePluginOpenAI enables AI features powered by OpenAI.
 	FeaturePluginOpenAI FeatureType = "bb.feature.plugin.openai"
+
+	// Execution.
+
+	// FeatureChunkedSubmit enables chunked submit.
```

Comment:
Since it's a release guard instead of feature guard, we can do simple branching checks along with the code.